_episodes/02-setup.md

```diff
@@ -73,7 +73,41 @@ For this lesson, we will be interacting with [GitHub](https://github.com/) and s
 > $ git config --global core.autocrlf true
 > ~~~
 > {: .language-bash}
-> 
+>
+{: .callout}
+
+> ## Default Pull Behavior
+>
+> As of `git` version 2.27 in order to avoid a distracting hint from being
```

Comment:
For this lesson we will use this default pull setting. Please enter:

Reply:
Sounds reasonable.

Reply:
Looks good to me!